src/components/App/AppSagas.js

```diff
@@ -41,7 +44,8 @@ export function* appInit() {
     console.info('User is signed in');
     yield loadApp();
   } else {
-    yield takeEvery(AUTH_SUCCESS_ACTION, loadApp),
-      yield setPageAction(PAGES_ENUM.LOGIN);
+    yield takeEvery(AUTH_SUCCESS_ACTION, loadApp);
+    // yield put(push('/'));
+    // yield setPageAction(PAGES_ENUM.LOGIN);
```

Comment:
Let's remove this since we're using router now.

Reply:
Please don't concentrate on refactoring and commented code. You can just checkout to merging branch and start app to see if everything working